docs/en_US/SearchSpaceSpec.md

```diff
@@ -36,9 +36,9 @@ All types of sampling strategies and their parameter are listed here:
     - Anneal
     - Evolution
 
-* {"_type":"randint","_value":[upper]}
+* {"_type":"randint","_value":[lower, upper]}
```

Comment:
What  will happen if a user already have a search space using old randint? such as  {"_type":"randint","_value":[100]} ?
will the trial still can run?

Reply:
we cannot support old randint now.

Reply:
Got it,  if the existing trial with old randint can not run anymore,  we may want to describe this change on our release note.

Reply:
how about backward compatibility? if users only write `upper`, the default value of `lower` is 0.

Reply:
further more, randint support step could be much better,  
`
"_type": "randint"
"_value": [10, 20, 2]
`